spec/models/ip_block_spec.rb

```diff
@@ -3,7 +3,42 @@
 require 'rails_helper'
 
 describe IpBlock do
-  describe 'to_log_human_identifier' do
+  shared_context 'with cache setup' do
+    let(:memory_store) { ActiveSupport::Cache.lookup_store(:memory_store) }
+    let(:cache) { Rails.cache }
+
+    before do
+      allow(Rails).to receive(:cache).and_return(memory_store)
+      Rails.cache.clear
+    end
+  end
```

Comment:
I don't think this is necessary, the test environment already sets up a memory store, and none of our tests explicitly set up a cache store.

Reply:
I totally missed the existing test environment setup. I just removed the unnecessary setup in 7793529e1ef3905a58e6ef5518b7af4243a171c0.